shell/browser/javascript_environment.cc

```diff
@@ -167,6 +167,11 @@ JavascriptEnvironment::~JavascriptEnvironment() {
   }
   isolate_->Exit();
   g_isolate = nullptr;
+
+  DCHECK_NE(platform_, nullptr);
+  platform_->DrainTasks(isolate_);
```

Comment:
Since we are running tasks, should this be moved before the context and isolate are exited.

Reply:
Force pushed. PTAL